javaharness/java/arcs/api/UiRenderer.java

```diff
@@ -1,5 +1,6 @@
 package arcs.api;
 
 public interface UiRenderer {
+
```

Comment:
nit - here and in a few other classes the only changes are empty lines. could you, please, revert them, it'll keep the change history cleaner.

Reply:
Done.